R/pearson.R

```diff
@@ -1208,11 +1208,11 @@ posteriorBetaParameters <- function(n, r, kappa=1, oneThreshold=1e-3) {
   # Let rho = 2*x - 1 where x \sim beta, thus, x = (rho+1)/2.Hence, someMu.
   # For the variance we have var(rho)/2^2
   #
-  someMu <- tryOrFailWithNA((posteriorMean("n"=n, "r"=r, "kappa"=kappa, "oneThreshold"=oneThreshold)+1)/2, )
+  someMu <- tryOrFailWithNA((posteriorMean("n"=n, "r"=r, "kappa"=kappa, "oneThreshold"=oneThreshold)+1)/2)
   someVar <- tryOrFailWithNA(posteriorVariance("n"=n, "r"=r, "kappa"=kappa, "oneThreshold"=oneThreshold)/4)
 
 
-  if (isSomeNA(someMu, someVar) || isSomeInfinite(someMu, someVar)) {
+  if (isSomeNA(someMu, someVar) || is.infinte(someMu) || is.infinte(someVar)) {
```

Comment:
@AlexanderLyNL are you sure this should be `is.infinte` and not `is.infinite`? :sweat_smile: 
```suggestion
  if (isSomeNA(someMu, someVar) || is.infinite(someMu) || is.infinite(someVar)) {
```

Reply:
Night work. Thanks for the catch.